src/sentry/integrations/pipeline.py

```diff
@@ -52,25 +52,26 @@ class IntegrationPipeline(Pipeline):
     def finish_pipeline(self):
         data = self.provider.build_integration(self.state.data)
         response = self._finish_pipeline(data)
+        self.provider.post_install(self.integration, self.organization)
         self.clear_session()
         return response
 
     def _finish_pipeline(self, data):
         if 'reinstall_id' in data:
-            integration = Integration.objects.get(
+            self.integration = Integration.objects.get(
                 provider=self.provider.key,
```

Comment:
I just have a question here. Does adding the provider here speed up the query any? I see you're already using the primary key in this query.

Reply:
Probably not, but I didn't add this, heh. Also, now that I'm looking at this, I think maybe the next line was intended to be `external_id=data['reinstall_id']`?